src/layouts/Layout.astro

```diff
@@ -7,34 +10,84 @@ const { title } = Astro.props;
 ---
 
 <!DOCTYPE html>
-<html lang="en">
+<html lang={i18next.language}>
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width" />
     <link rel="icon" type="image/svg+xml" href="/favicon.svg" />
     <meta name="generator" content={Astro.generator} />
+    <HeadHrefLangs />
     <title>{title}</title>
   </head>
   <body>
+    <header>
+      <LanguageSelector showFlag={true} class="language-selection" />
+    </header>
     <slot />
+    <style is:global>
```

Comment:
I believe the `<style>` tag should always be outside of the HTML in Astro. At least it looks like this in the [docs](https://docs.astro.build/de/guides/styling/), and Astro also generated the project with `<style>` outside of `<html>`.

Reply:
My formatter used to move it inside the html tag. Sry.